tools/ds-identify

```diff
@@ -1558,6 +1558,11 @@ dscheck_NWCS() {
     return ${DS_NOT_FOUND}
 }
 
+dscheck_Aeza() {
+    dmi_sys_vendor_is Aeza && return ${DS_FOUND}
```

Comment:
Given that we check sys vendor id for Aeza as a match for the datasource, in ds-identify, this same detection logic should be performed in a `DataSourceAeza.ds_detect` method in python which returns a boolean True if vendor = dmi.read_dmi_data("system-manufacturer") == "Aeza"

Reply:
Fixed in 4350dc221feb40ba2e277706e890126f92cee282

Reply:
Thank you, looks good!